code/drasil-gen/stack.yaml

```diff
@@ -34,7 +34,9 @@ packages:
 - ../drasil-build
 - ../drasil-code
 - ../drasil-code-base
+- ../drasil-data
```

Comment:
why did drasil-gen gain these dependencies?

Reply:
It was a bug that it wasn't there (if you were in `drasil-gen` and had run `stack build`, it would've failed due to a missing dep). One of the packages it relies on depends on drasil-data somewhere.